<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>etudiantDialog</class>
 <widget class="QDialog" name="etudiantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>631</width>
     <height>611</height>
    </rect>
   </property>
   <property name="title">
    <string>Etudiant</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ENREGISTRER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTRER LE NUMERO DE L'ETUDIANT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTRER LE NOM :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTRER LE PRENOM:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTRER LA SPECIALITE:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IIR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IFA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AI</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../student.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
